lib/src/execute.rs

```diff
@@ -75,9 +75,7 @@ impl ExecuteCtx {
         let mut linker = Linker::new(&engine);
         link_host_functions(&mut linker, &wasi_modules)?;
         let module = Module::from_file(&engine, module_path)?;
-
-        let mut dummy_store = dummy_store(&engine);
```

Comment:
:pray: i'm particularly excited to see the `Store` requirement here dropped. this is very nice!